### ISSUE TITLE
Sort on reload issues

### ISSUE THREAD

**Issue by janosimas** [VioletGiraffe/cppcheck-vs-addin#189]
After sorting the issues list:
if the list is reloaded the 'sort arrow' in the tab name is still there but the issues are not sorted.

**Comment by VioletGiraffe** [comment]
Yeah, they're added in the order that `cppcheck` spits them out regardless of the last selected sort order, I think. More an issue of the `ListView` not sorting new items automatically, but I can certainly trigger a re-sort manually. However, then there is no way to view the issues in their natural order (short of adding a number/index column).
